lib/Slic3r/GUI/3DScene.pm

```diff
@@ -744,7 +745,15 @@ sub InitGL {
     # Set antialiasing/multisampling
     glDisable(GL_LINE_SMOOTH);
     glDisable(GL_POLYGON_SMOOTH);
-    glEnable(GL_MULTISAMPLE);
+
+    # See "GL_MULTISAMPLE and GL_ARRAY_BUFFER_ARB messages on failed launch"
```

Comment:
I don't know what all this "junk" is for, my PR is cleaner and just contains the essentials : #4085